qiskit/providers/aer/noise/utils/noise_transformation.py

```diff
@@ -691,22 +692,13 @@ def solve_quadratic_program(self, P, q):
         Returns:
             list: The solution of the quadratic program (represents probabilites)
 
-        Raises:
-            ImportError: If cvxopt external module is not installed
-
         Additional information
         ======================
-        This method is the only place in the code where we rely on the cvxopt library
+        This method is the only place in the code where we rely on the cvpy library
         should we consider another library, only this method needs to change
         """
-        try:
-            import cvxopt
```

Comment:
I think we should leave `import cvxpy` here so that it isn't imported by the top level qiskit aer init.

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/430/commits/426bcfccc69684c4ba50848b4ec8605ca59ad78b